automated-comments/csharp/general/convert_variable_to_const.md

```diff
@@ -0,0 +1 @@
+Consider converting the `%{name}` variable to a `const`, as the value is intended never to change. Usually, `const` values are defined at the class level, as they are frequently used in multiple methods. In this case, the `const` value can be defined within the method itself, which means that other methods cannot use its value.
```

Comment:
Will the analyser still block if this change is not made? Or is this always a "tip"? If it still blocks:


```suggestion
Convert the `%{name}` variable to a `const`, as the value is intended never to change. Usually, `const` values are defined at the class level, as they are frequently used in multiple methods. In this case, the `const` value can be defined within the method itself, which means that other methods cannot use its value.
```